Artifacts.toml

```diff
@@ -406,13 +406,13 @@ os = "windows"
 
 [["micromamba-0.27.0"]]
 arch = "x86_64"
-git-tree-sha1 = "d45b66605cad8bca223a3eb103eebb71adcf1415"
+git-tree-sha1 = "f7f4588f4cdb2a84682d91f52ffe5eb98242a4cb"
 lazy = true
 libc = "glibc"
 os = "linux"
 
     [["micromamba-0.27.0".download]]
-    sha256 = "4adbf3091a4159af2c48264a8e32ecb98147b0e3f200601f384f8f53a6910ca2"
+    sha256 = "1b1bdd0798c3c57eda3c76660ed148a59d70e555c6dcd3c099e824bf78873d70"
     url = "https://micro.mamba.pm/api/micromamba/linux-64/0.27.0"
```

Comment:
The problem is that this URL is not reliable.  I _think_ URLs like https://api.anaconda.org/download/conda-forge/micromamba/0.27.0/linux-64/micromamba-0.27.0-0.tar.bz2 are more stable, because they include the build number.

Reply:
I agree it's not ideal. My hope is that we can switch over to jlls soon enough and this will tide things over in the meantime.

Reply:
My point is that you should change the URL instead of the sha256 (and not just for this platform, but for all the other 0.27.0 builds):
```console
% wget -qO - 'https://api.anaconda.org/download/conda-forge/micromamba/0.27.0/linux-64/micromamba-0.27.0-0.tar.bz2' | sha256sum -
4adbf3091a4159af2c48264a8e32ecb98147b0e3f200601f384f8f53a6910ca2  -
```
Also because you have also to update the tree hash anyway, the PR as it is now is broken.

Reply:
FYI, a simple and robust way to get the list of packages is to regularly fetch https://conda.anaconda.org/conda-forge/linux-64/repodata.json and look for packages with `name: micromamba`:

```json
...
    "micromamba-0.27.0-2.tar.bz2": {
      "build": "2",
      "build_number": 2,
      "depends": [],
      "license": "BSD-3-Clause AND MIT AND OpenSSL",
      "license_family": "BSD",
      "md5": "1302c14c1f77ae7964824355778fac69",
      "name": "micromamba",
      "sha256": "1b1bdd0798c3c57eda3c76660ed148a59d70e555c6dcd3c099e824bf78873d70",
      "size": 6144724,
      "subdir": "linux-64",
      "timestamp": 1666874250676,
      "version": "0.27.0"
    },
...
```